### DISCUSSION TITLE
Ratelimit all requests

### DISCUSSION THREAD

**Discussion by forforeach** [jsocol/django-ratelimit#233] [Q&A] [answered]
Hello, thanks for the project work you do!

I'm trying to understand if there is a way to define a single rate limit definition to all requests. I checked the middleware, but it looks like it only handles the rate limit exceptions

**Comment by adamchainz** [comment, marked as answer]
You can make your own middleware that calls `is_ratelimited` for each request.

**Reply by jsocol** [reply]
Yep, this would be the easiest way. Set the `group=` kwarg to a constant, and all requests will fall into the same group, so they'll be counted together for a particular value.

Also, thanks for using discussions to ask! I'm excited about trying this out instead of only having issues.

**Reply by forforeach** [reply]
Yeah, that's what I thought about. Thanks for the quick answer. 

Also, the discussions feature of GitHub is very cool. Finally, one can ask a question and not filing a new issue.
